src/openrct2/actions/PlacePeepSpawnAction.cpp

```diff
@@ -47,12 +47,6 @@ GameActions::Result::Ptr PlacePeepSpawnAction::Query() const
     res->Expenditure = ExpenditureType::LandPurchase;
     res->Position = _location;
 
-    if (!map_check_free_elements_and_reorganise(3))
```

Comment:
peep spawns dont use tile elements so this check was pointless. Likely copied from the park entrance code.